docs/csi-debug.md

```diff
@@ -0,0 +1,136 @@
+## CSI driver troubleshooting guide
+
+&nbsp;
+
+### Case#1: volume create/delete issue
+
+&nbsp;
+
+- Locate csi driver pod
+
+```console
+$ kubectl get po -o wide -n kube-system | grep csi-azurelustre-controller
+```
+
+<pre>
+NAME                                              READY   STATUS    RESTARTS   AGE     IP             NODE
+csi-azurelustre-controller-56bfddd689-dh5tk       3/3     Running   0          35s     10.240.0.19    k8s-agentpool-22533604-0
+csi-azurelustre-controller-56bfddd689-sl4ll       3/3     Running   0          35s     10.240.0.23    k8s-agentpool-22533604-1
+</pre>
+
+&nbsp;
+
+- Get csi driver logs
+
+```console
+$ kubectl logs csi-azurelustre-controller-56bfddd689-dh5tk -c azurelustre -n kube-system > csi-lustre-controller.log
+```
+
+> note: there could be multiple controller pods, logs can be taken from all of them simultaneously, also with `follow` (realtime) mode
```

Comment:
Perhaps we could optimize all this work into a single kubectl logs command like:
kubectl logs -nkube-system -lapp=csi-azurelustre-controller -cazurelustre --tail=-1 --prefix >csi-lustre-controllers.log